README.md

```diff
@@ -55,37 +55,64 @@ Alternatively you can wrap any code which may throw errors using the client's `w
 
 ## Advanced Usage
 
-### Default Annotations
+### Notice Annotations
 
-It's possible to annotate error notices with all sorts of useful information. Below, the various top-level interface methods are listed, along with their effects.
-
-* `airbrake.setEnvironmentName(string)` sets the environment name, e.g. production or staging.
-* `airbrake.addEnvironment(object)` adds environment information reported alongside all errors.
-* `airbrake.addParams(object)` adds params information reported alongside all errors.
-* `airbrake.addSession(object)` adds session information reported alongside all errors.
-* `airbrake.addContext(object)` adds context information reported alongside all errors. This hash is reserved for notifier and Airbrake backend will ignore unknown keys.
-
-Additionally, much of this information can be added to captured errors at the time they're captured by supplying it in the object being reported.
+It's possible to annotate error notices with all sorts of useful information at the time they're captured by supplying it in the object being reported.
 
     try {
       startApp();
     } catch (err) {
-      airbrake.push({
+      airbrake.notify({
```

Comment:
In Ruby we use slightly different parameters. I suppose we want to keep the old way of passing params in order not to break the compliance with the old `airbrake.push`?

In Ruby we can only pass `params`, which looks like this: `airbrake.notify(err, foo: 'bar')`, where `foo: 'bar'` is `params: {foo: 'bar'}`. We allow modify other payload only via filters there.